core/services/functions/listener_export_test.go

```diff
@@ -0,0 +1,11 @@
+package functions
+
+import "github.com/ethereum/go-ethereum/common"
+
+func (l *FunctionsListener) VerifyRequestSignature(requestID RequestID, subscriptionOwner common.Address, requestData *RequestData) error {
```

Comment:
Not a huge fan of this. Maybe you could put VerifyRequestSignature() in request.go?